test/blackbox-tests/test-cases/github11527.t/dune-project

```diff
@@ -0,0 +1,4 @@
+(lang dune 3.17)
+
+(package
+ (name bug))
```

Comment:
I think the `package` can be left out and would make the test better as it doesn't imply that it has something to do with packages?